src/dbt_score/scoring.py

```diff
@@ -19,6 +20,11 @@ class Score:
     value: float
     badge: str
 
+    @property
+    def human_value(self) -> float:
```

Comment:
Clarified that this is only for human-readable purpose (instead of naming it e.g. `rounded_value`)

Reply:
IMO it's clearer to use names that relate to what it is, rather than the intended use. That would make it more descriptive, and suitable for other usages. No strong opinion though

Reply:
Would you have a more suitable suggestion for a name?

Reply:
I think `rounded_value` as you mentioned above, or `floored_value`, would work well